tools/osx/macosx_bundle.sh

```diff
@@ -127,6 +127,19 @@ ditto ${LOCAL_PREFIX}/local/lib/libomp.dylib "${CONTENTS}/Frameworks"
 msg "Copying dependencies from ${GTK_PREFIX}:"
 CheckLink "${EXECUTABLE}"
 
+# dylib install names
+find -E "${CONTENTS}" -type f -regex '.*/(rawtherapee-cli|rawtherapee|.*\.(dylib))' | while read -r x; do
+    msg "Modifying dylib install names: ${x}"
+    {
+        # id
+        case ${x} in *.dylib) echo "   install_name_tool -id '@rpath/$(basename "${x}")' '${x}'";; esac
```

Comment:
An `if` would be better suited than a `case`.

Reply:
Thanks I will convert that tonite.

Reply:
changed to `if` and made it a function.